pkg/controller/ehpa/substitute_controller.go

```diff
@@ -61,24 +60,13 @@ func (c *SubstituteController) Reconcile(ctx context.Context, req ctrl.Request)
 		c.Log.Info("Update Substitute status successful", "Substitute", klog.KObj(substitute))
 	}
 
-	// Rsync every 1 minute
+	// Rsync every 15 seconds
```

Comment:
1. is 15s too fast? if there are 5000 subs, then the controller will be busy on reconcile almost all the time.
2. can we make this a customize option with 1min default interval?

Reply:
The background is controller sync not only selectors here but also replicas, since HPA controller reconcile hpa every 15 seconds by default, so we may follow the 15 seconds here.
This can be a start options.